practice_projects/#Homework_pt.py

```diff
@@ -0,0 +1,34 @@
+#Homework pt.1
+
+Friend= "\nSoftware is my friend!\n"
```

Comment:
Nitpick:
Space on each side of equals sign:
`Friend = "\nSoftware is my friend!\n"`

Reply:
fixed